src/main/java/reactor/core/publisher/FluxCreate.java

```diff
@@ -114,23 +115,23 @@ public void subscribe(Subscriber<? super T> t) {
 
 		volatile boolean done;
 
-		public SerializedSink(BaseSink<T> sink) {
+		SerializedSink(BaseSink<T> sink) {
 			this.sink = sink;
-			this.queue = QueueSupplier.<T>unbounded(QueueSupplier.XS_BUFFER_SIZE).get();
+			this.queue = QueueSupplier.<T>unbounded(16).get();
```

Comment:
yeah, my point was that we avoid magic numbers everywhere else. using `XS_BUFFER_SIZE` has the advantage that this can be tuned if it becomes necessary (eg. server with large number of cores, etc...)